flink-connectors/flink-sql-connector-elasticsearch6/pom.xml

```diff
@@ -36,11 +36,65 @@ under the License.
 	<packaging>jar</packaging>
 
 	<dependencies>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-elasticsearch-base_${scala.binary.version}</artifactId>
+			<version>${project.version}</version>
+			<exclusions>
+				<!-- Elasticsearch Java Client has been moved to a different module in 5.x -->
+				<exclusion>
+					<groupId>org.elasticsearch</groupId>
+					<artifactId>elasticsearch</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+
 		<dependency>
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-connector-elasticsearch6_${scala.binary.version}</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+
```

Comment:
These should now be removed from the pure ES6 connector because they are not needed anymore.

Reply:
In my memory, I used to try to remove these dependencies in the PR #9720 , but this behavior caused test case failure, so I gave up, let me try again.